Gemfile

```diff
@@ -3,6 +3,9 @@ ruby IO.read('.ruby-version').strip
 source 'https://rails-assets.org' do
   gem 'rails-assets-listjs'
   gem 'rails-assets-zloirock--core-js'
+  gem 'rails-assets-fullcalendar'
+  gem 'rails-assets-fullcalendar-scheduler'
+  gem 'rails-assets-qTip2'
```

Comment:
Probably not an issue, but qTip isn't used here at all, but will be used once we get the company calendar in there....leave it up to you whether we leave it in or not.